src/ui/Folding/Folding.ts

```diff
@@ -334,29 +352,45 @@ export class Folding extends Component {
       if (resultNodes[i].result.uniqueId == uniqueId) {
         return resultNodes[i];
       }
-      let resultNode = Folding.findUniqueId(resultNodes[i].children, uniqueId);
+      const resultNode = Folding.findUniqueId(resultNodes[i].children, uniqueId);
       if (resultNode != null) {
         return resultNode;
       }
     }
     return null;
   }
 
+  private switcherooFoldingFields() {
```

Comment:
I think the name could be more specific, maybe `swapParentChildFoldingFields`?

Reply:
I agree; this does not look like clean code.
Sami's suggestion is good I feel.

Reply:
`switcherooooo`